blazor/speeddial/positions.md

```diff
@@ -36,94 +65,45 @@ The position values of Speed Dial are as follows:
-[HideAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_HideAsync) methods.
-
-Below example demonstrates open/close action items on button click.
-
-```cshtml
-
-@using Syncfusion.Blazor.Buttons
-
-<div id="target" style="height:200px; position:relative; width:300px; border:1px solid;">
-    <SfButton OnClick="Show">Show</SfButton>
-    <SfButton style="float:right"  OnClick="Hide">Hide</SfButton>
-    <SfSpeedDial @ref="speeddial" Target="#target" Position="FabPosition.BottomCenter" OpenIconCss="e-icons e-edit" CloseIconCss="e-icons e-close">
-        <SpeedDialItems>
-            <SpeedDialItem IconCss="e-icons e-cut"/>
-            <SpeedDialItem IconCss="e-icons e-copy"/>
-            <SpeedDialItem IconCss="e-icons e-paste"/>
-        </SpeedDialItems>
-    </SfSpeedDial>
-</div>
+<style>
 
-@code{
-    SfSpeedDial speeddial;
-    public void Show()
-    {
-        speeddial.ShowAsync();
-    }
-    public void Hide()
-    {
-        speeddial.HideAsync();
+    .e-fab.e-btn.custom-position {
+        left: 325px;
+        top: 170px;
+        bottom: unset;
+        right: unset;
     }
-}
-
-```
-
-![Blazor Speed Dial Show Items](./images/Blazor-SpeedDial-ShowItem.png)
-![Blazor Speed Dial Hide Items](./images/Blazor-SpeedDial-HideItem.png)
 
-## Programmatically refresh the position
-
-You can refresh the position of the Speed Dial using [RefreshPositionAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_RefreshPositionAsync) method when the `Target`position is changed.
-
-```cshtml
+    .e-speeddial-popup.e-speeddial-right.e-speeddial-linear {
+        left: 315px;   
+    }
 
-@using Syncfusion.Blazor.Buttons
+    .e-speeddial-popup.e-speeddial-bottom.e-speeddial-linear {
+        top: 50px;
+    }
 
-<div id="target" style="min-height:350px; position:relative; border:1px solid;">
-    <SfButton OnClick="RefreshPosition">Refresh</SfButton>
-    <SfSpeedDial @ref="speeddial" Target="#target" Position="FabPosition.MiddleRight" OpenIconCss="e-icons e-edit" CloseIconCss="e-icons e-close">
-        <SpeedDialItems>
-            <SpeedDialItem IconCss="e-icons e-cut"/>
-            <SpeedDialItem IconCss="e-icons e-copy"/>
-            <SpeedDialItem IconCss="e-icons e-paste"/>
-        </SpeedDialItems>
-    </SfSpeedDial>
-</div>
+</style>
 
-@code{
-    SfSpeedDial speeddial;
+```
 
-    public void RefreshPosition()   
-    {
-        speeddial.RefreshPositionAsync();
-    }
-}
+![Blazor SpeedDial component with custom position](./images/Blazor-SpeedDial-Custom-Position.png)
 
-```
\ No newline at end of file
+N> You can use the [RefreshPositionAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_RefreshPositionAsync) method when the `Target`position is changed or resized.
```

Comment:
N> You can use the [RefreshPositionAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_RefreshPositionAsync) method when the `Target`position is resized.

Reply:
Modified the content as given